SUMMARY.md

```diff
@@ -0,0 +1,8 @@
+# Table of contents
+
+- [Introduction](README.md)
+
```

Comment:
We could also add a section for tokenonmics and the AMM mechanics.

Reply:
Yes for sure. We can do this in the next PRs. This PR is just to get started small. I created already quite some confusion and having bigger PRs just creates more confusion at a time.

Reply:
Good stuff, let's do this.